pages/documentation/simulation/powerflow/openlf.md

```diff
@@ -10,6 +10,46 @@ PowSyBl OpenLoadFlow is an open-source power flow implementation in Java provide
 * TOC
 {:toc}
 
+## Grid modelling
+
+OpenLoadFlow computes power flows from IIDM grid model in bus/view topology. From the view, a very simple network, composed of only buses and branches is created. In the graph vision, we rely on a `Pi` model for branches (lines, transformers, dangling lines, etc.):
+
+- `R` and `X` are respectively the real part (resistance) and the imaginary part (reactance) of the complex impedance ;  
+- `G1` and `G2` are the real parts (conductance) on respectively side 1 and side 2 of the branch ;
+- `B1` and `B2` are the imaginary parts (susceptance) on respectively side 1 and side 2 of the branch ;
+- `A1` is the angle shifting on side 1, before the series impedance. For classical branches, the default value is zero ;
+- `R1` is the ratio of voltages between side 2 and side 1, before the series impedance. For classical branches, the default value is `1`.
+
+As the `Pi` model is created from IIDM grid modelling that locates its ratio and phase tap changers in side 1, `A2` and `R2` are always equal to zero and `1`. In case of a branch with voltage or phase control, the `Pi` model becomes an array. See below our model:
+
+![Pi model](img/pi-model.svg){: width="50%" .center-image}
+
+### AC flows computing
+
+TO DO
+
+### DC flows computing
+
+The DC flows computing relies on several classical assumptions to build a model where the active power flowing through a line depends linearly from the voltage angles at its ends.
+In this simple model, reactive power flows and active power losses are totally neglected. The following assumptions are made to ease and speed the computation:
+- The voltage magnitude is equal to 1 per unit at each bus,
+- The conductance \\(G_{i,j}\\) of each line \\((i,j)\\) is neglected, only the susceptance \\(B_{i,j}\\) is considered,
+- The voltage angle difference between two adjacent buses is considered as very small.
+
+Therefore, the power flows from bus \\(i\\) to bus \\(j\\) following the linear expression:
+
+$$ P_{i,j} = \frac{\theta_i-\theta_j+\phi_{i,j}}{X_{i,j}} $$
+
+Where \\(X_{i,j}\\) is the reactance of the line \\((i,j)\\), and \\(\theta_i\\) the voltage angle at bus \\(i\\).
+
+The first step to compute the sensitivities is to build the grid constraints linear system. The variables of the system are, for each bus, the voltage angle \\(\theta\\). The constraints of the system are the active power balance at each bus, except for the slack bus. The voltage angle at slack bus is set to zero. Therefore the linear system is composed of \\(N\\) variables and \\(N\\) constraints, where \\(N\\) is the number of buses in the network:
```

Comment:
Can you explicit the right hand side ? I think it is useful to understand sensitivity analysis later.

Reply:
Done in cf112685569d166d678807236176641dd11e2b0b